guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Resolving_Package_Dependencies_{context}"]
+= Resolving Package Dependencies
+
+In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
+
+You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
```

Comment:
Is "enabled" or "disabled" the default setting? Either way, IMHO you should remove one aka. "or you can change the default setting to disable resolving package..." or "or you can change the default setting to enable resolving package...".

Reply:
> Is "enabled" or "disabled" the default setting?
disable seems to be the default. changing the wording appropriately